assets/src/scss/pages/_page.scss

```diff
@@ -121,3 +121,80 @@
     outline: 0 none;
   }
 }
+
+.bottom-navigation {
+  height: 80px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 15px;
+  font-weight: var(--font-weight-bold);
+  font-family: var(--font-family-heading);
+  font-size: var(--font-size-xxs--font-family-primary);
+
+  a {
+    display: flex;
+    align-items: center;
+  }
+
+  a:hover {
+    text-decoration: none;
+  }
+
+  &-next {
+    margin-inline-start: auto;
+
+    &::after {
+      background-image: url("../../images/arrow_forward.svg");
+      margin-inline-start: $sp-x;
+    }
+  }
+
+  &-prev {
+    &::before {
+      background-image: url("../../images/arrow_back.svg");
+      margin-inline-end: $sp-x;
+    }
+  }
+
+  @include small-and-less {
+    a {
+      width: 50%;
+    }
+
+    .bottom-navigation-link-text {
```

Comment:
This doesn't work for sub-items, for example on the test instance for the children of `The issues we work on`